llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp

```diff
@@ -52,6 +52,8 @@ namespace {
 static VNInfo *getVNInfoFromReg(Register Reg, const MachineInstr &MI,
```

Comment:
Nit, should probably update the comment to mention it will return nullptr if LIS is null too